### ISSUE TITLE
torchaudio.transformes.RNNTLoss

### ISSUE THREAD

**Issue by pipichen73** [pytorch/audio#2196]
### 🐛 Describe the bug

If I don't use the max seq length for logit length, it will cause run time error, input length mismatch.

def compute_loss(self, joiner_output: Tensor, enc_output_mask: Tensor, tgt_seq: Tensor):
        """compute rnnt loss 
        Args:
            joiner_output: (Tensor): B X T X (U+1) X CLASSES
            enc_output_mask (Tensor): B X T
            tg_seq (Tensor): B X U
        Returns:        
        """
        logit_length = enc_output_mask.sum(dim=-1).to(torch.int32) #torch.full((joiner_output.shape[0],), fill_value=joiner_output.shape[1]).to(torch.int32) #
        target_length = tgt_seq.gt(0).sum(dim=-1).to(torch.int32)
        # print("logit {} {} target {} {}".format(joiner_output.shape, logit_length.shape, tgt_seq.shape, target_length.shape))
        # print("len {} targetlen{}".format(logit_length, target_length))
        loss = self.rnnt_loss(logits=joiner_output, targets=tgt_seq,
             logit_lengths=logit_length, target_lengths=target_length)
        
        return loss

### Versions

torch                  1.10.2
torchaudio             0.10.2
torchvision            0.10.0

**Comment by nateanl** [comment]
Hi @pipichen73, thanks for posting the issue. To figure out the cause, could you paste an example that will throw the runtime error?

**Comment by pipichen73** [comment]
Hi,
I found the problems.
The tensors i passed in had issues.


Thanks for your help.

Hui

**Comment by nateanl** [comment]
Great that you found the cause! I'm going to close this issue, but let me know if you meet any further issue.
